modules/3d-tiles/src/tileset/tileset-3d.js

```diff
@@ -387,6 +417,18 @@ export default class Tileset3D {
     // this._readyPromise.resolve(this);
   }
 
+  _initStats() {
+    this.stats.get(TILES_TOTAL);
+    this.stats.get(TILES_LOADING);
+    this.stats.get(TILES_IN_MEMORY);
+    this.stats.get(TILES_IN_VIEW);
+    this.stats.get(TILES_RENDERABLE);
+    this.stats.get(TILES_LOADED);
+    this.stats.get(TILES_UNLOADED);
+    this.stats.get(TILES_LOAD_FAILED);
+    this.stats.get(POINTS_COUNT);
```

Comment:
I would like that the `stats` component not does need users to do the initializtaion.

Reply:
My assumption is that it would still work if I didn't initialize them here, but the stats won't show up in the widget until they are used, and the order of the stats in the widget would be more "random", based on which one was first modified etc.